### ISSUE TITLE
UX: VBA Regional Office Events

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#17666]
## Background
Outreach Hub and VAMC editors have access to two content types to feature events on their pages: Event, and Events list. 
Events are a very important way that Vet Centers and Regional benefit offices provide services. We want to enable this functionality for their facility pages but remain blocked by content build time. 

We should be unblocked in the Spring of 2024 by the AP effort or a retool of the approach to events. In the meanwhile, we can prepare for implementation from a design perspective. 

## Background
<details>
<summary> Existing event functionality </summary>

- The Event content type allows editors to create, cancel, or reschedule events
- Events exist as individual events and are added to an events list page
- Events can be featured on the health care system homepage
- Event details include start and end date/time with optional recurrence as daily, weekly, or monthly with a recurrence end date until a year into the future.
- Event locations can be specified at a VA facility via dropdown, a non-VA location by adding the address, or online 
- Events can be shared to a global VA.gov events page
- Events can include an event image and registration info

</details>

<details>
<summary> Example VAMC and benefit hub event pages </summary>

- [Event listing page for Houston health care](https://www.va.gov/houston-health-care/events/)

- [Legal Assistance for Veterans and Dependents event page](https://www.va.gov/houston-health-care/events/61545)

- [Outreach hub event listing](https://www.va.gov/outreach-and-events/events/)

</details>

<details>
<summary> Example featured event on Houston health care home page </summary> 

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbWZHIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--c3581202f014fdc431166fe80c142d108164a202/featured%20event.jpeg)

</details>

### Regional office events 

- [Honolulu Regional Office](https://www.benefits.va.gov/honolulu/)
- [San Diego Regional Office](https://www.benefits.va.gov/sandiego/)

## Open questions
- Will the existing event functionality meet Regional benefit office needs? 
- Imagine events for all facility types are tagged to make filtering possible, such as event type (in person vs virtual, event topic (PACT Act vs Education vs Pension), or event audience (family member vs Veteran vs public). If this were possible, what types of filters might be needed for VBA events?  

## Problems to solve
The design should consider the following scenarios from a Veteran-facing perspective
- MVP VBA pages have no left nav menu. How might we make events discoverable on these pages? 
   - How might we support expansion of these pages (i.e. support for other location types)? 
- Imagine events for all facility types are tagged to make filtering possible. What is the Veteran experience? How might a Veteran or other audience search for the events they are interested in? How might they learn more about a particular event? 
- Events occur in a wide range of  time zones, including locations in Guam and the Philippines
- How might an image be added to an event listing? 

Support for multiple facility photos images is captured in a separate issue

## Tasks (Proposed - TBD)
- Discovery with existing RO and Vet Center staff/editors to understand use cases and needs
- IA input
- Menu/Page design
- ??
